docs/src/modules/java/pages/actions-publishing-subscribing.adoc

```diff
@@ -0,0 +1,46 @@
+= Publishing and Subscribing with Actions
+
+A very common use case when building Microservices is to publish and subscribe to a stream of events. 
+
+With Actions you can: 
+* subscribe to events emitted by an entity within the same service.
```

Comment:
do we need to clarify that it's also possible to subscribe to state changes from value entities?

Reply:
yeah, I think so. I will expand.